Cabal/Distribution/PackageDescription/FieldGrammar.hs

```diff
@@ -539,3 +542,16 @@ setupBInfoFieldGrammar def = flip SetupBuildInfo def
     <$> monoidalFieldAla "setup-depends" (alaList CommaVCat) L.setupDepends
 {-# SPECIALIZE setupBInfoFieldGrammar :: Bool -> ParsecFieldGrammar' SetupBuildInfo #-}
 {-# SPECIALIZE setupBInfoFieldGrammar :: Bool ->PrettyFieldGrammar' SetupBuildInfo #-}
+
+-------------------------------------------------------------------------------
+-- Script
+-------------------------------------------------------------------------------
+
+scriptFieldGrammar
+    :: (FieldGrammar g, Applicative (g Script), Applicative (g Executable), Applicative (g BuildInfo))
+    => g Script Script
+scriptFieldGrammar = Script
+    <$> blurFieldGrammar L.executable (executableFieldGrammar "script")
+    <*> optionalFieldAla "with-compiler" FilePathNT L.hcPath
+{-# SPECIALIZE scriptFieldGrammar :: ParsecFieldGrammar' Script #-}
+{-# SPECIALIZE scriptFieldGrammar :: PrettyFieldGrammar' Script #-}
```

Comment:
I don't really get why this pragma is used twice here. This does seem to be the case in the rest of the definitions above as well though, weird.

Reply:
Oh, I can't read. The first one is for "Parsec" and the other one for "Pretty". nvm.

Reply:
@DanielG indeed, I'm trying had to name dual things with nicely alignable names! :)